source/fundamentals/crud/read-operations/specify-query.txt

```diff
@@ -0,0 +1,353 @@
+.. _csharp-specify-query:
+
+===============
+Specify a Query
+===============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to specify a query using the {+driver-long+}.
+
+You can narrow the set of matched documents returned by your query by creating a
+**query filter**. A query filter is an expression that specifies the documents you 
+want to match in a read, update, or delete operation.
+
+.. note:: Using LINQ
+
+   This guide shows how to specify queries using query filters. You can also
+   specify queries using LINQ. To learn more about using LINQ, see
+   :ref:`csharp-linq`.
+
+The examples in this guide use the following documents in a collection called
+``guitars``:
+
+.. code-block:: json
+ 
+   { "_id": 1, "make": "Fender", "models": ["Stratocaster", "Telecaster"], "establishedYear": 1946, "rating": 9 }
+   { "_id": 2, "make": "Gibson", "models": ["Les Paul", "SG", "Explorer"], "establishedYear": 1902, "rating": 8 }
+   { "_id": 3, "make": "PRS", "models": ["Silver Sky", "SE", "Custom"], "establishedYear": 1985, "rating": 9 }
+   { "_id": 4, "make": "Kiesel", "models": ["Ares", "Vader", "Solo"], "establishedYear": 2015 }
+   { "_id": 5, "make": "Ibanez", "models": ["RG", "AZ"], "establishedYear": 1957, "rating": 7 }
+   { "_id": 6, "make": "Strandberg", "models": ["Boden", "Salen"], "establishedYear": 1982 }
+
+The following ``Guitar`` class models the documents in this collection.
+
+.. TODO: Add link to mapping page.
+
+.. literalinclude:: /includes/fundamentals/code-examples/specify-query/Guitar.cs
+   :language: csharp
+   :copyable:
+   :dedent:
+
+The following code instantiates the ``guitars`` collection with this class. The 
```

Comment:
Issue: The ``guitars`` collection is not created/instantiated through the following code snippet, but rather the ``_guitarsCollection`` object.

Suggestion: "with the class" is a bit unhelpful (it doesn't say what instantiating the class in this way). Consider using language like "serialize" / "deserialize":

```suggestion
The following code instantiates the ``Collection`` object ``_guitarsCollection`` using the ``Guitars`` class as a type parameter.  This type parameter  causes the ``_guitarsCollection`` object to automatically serialize and deserialize the BSON documents it receives from MongoDB to instances of the ``Guitars`` class: 
```